demo/mobile/src/app/sync/syncGlobalNetworkHandler.js

```diff
@@ -2,7 +2,9 @@ module.exports = function createHandler(cloudURL, cloudPath, $http) {
   var handler = function(params, success, failure) {
     var url = cloudURL + cloudPath + params.dataset_id;
     var payload = params.req;
-    $http.post(url, payload).then(function(response) {
+    $http.post(url, payload, {headers: {
+      'Authorization': 'JWT ' + localStorage.getItem('rctoken')
```

Comment:
That may be `JWT undefined` when token is gone. Best to send that only when is defined.